src/databases/Blueprint/avtBlueprintFileFormat.C

```diff
@@ -870,6 +928,83 @@ avtBlueprintFileFormat::ReadRootFile()
 
 }
 
+// ****************************************************************************
+//  Method: avtBlueprintFileFormat::ReadRootIndexItems
+//
+//  Purpose: Read only the metadata portions of the root file that VisIt needs.
+//
+//  Programmer: cyrush
+//  Creation:   Mon Mar  9 12:24:16 PDT 2020
+//
+//  Modifications:
+//
+//
+// ****************************************************************************
+void
+avtBlueprintFileFormat::ReadRootIndexItems(const std::string &root_fname,
+                                           const std::string &root_protocol,
+                                           conduit::Node &root_info)
+{
+    // list of names we want to read from the root file
+    conduit:Node index_names;
```

Comment:
Isn't there a shorthand, initialization for this as in

```
conduit:Node const index_names = "{
    blueprint_index,
    file_pattern,
    tree_pattern,
.
.
."};
```

Reply:
there isn't at this point, there are fancier possibilities embracing newer C++ features.

I'll admit the list case does look a little strange, however I chose it b/c its less typing than using a std::vector<std::string>.

Reply:
Hmm, well, I could use embedded json or yaml here -- thats a possibility. But I think I'll keep as is to avoid parsing.